frontend/src/renderer/util/autoSortStandards.svelte

```diff
@@ -0,0 +1,120 @@
+<script context="module">
+    import { maxBy, each, includes, size, filter, countBy, minBy } from "lodash";
+
+    const artObjectStandards = [
+        "image",
+        "print",
+        "object",
+        "art"
+    ];
+    const targetStandards = [
+        "target",
+        "color",
+        "colors",
+        "grid"
+    ];
+    const flatFieldStandards = [
+        "flat",
+        "field",
+        "white",
+        "blank"
+    ];
+    const darkFieldStandards = [
+        "dark",
+        "field",
+        "black"
+    ];
+
+    const probabilityScoreProperties = [
+        'artObjectProbability',
+        'targetProbability',
+        'flatFieldProbability',
+        'darkFieldProbability'
+    ];
+
```

Comment:
help me fix this

Reply:
This is definitely going to updated once user configs are a thing, but i think it should search for keywords with a delimiter (like surrounded by underscores or some other character) so that it doesnt pick out peoples names like they had mentioned. otherwise i dont have much of an idea for a better system

Reply:
Additionally, as per the requirements, the user has the opportunity to sort things manually after using auto-sort, so any issues with matching aren't really a detraction of any existing feature